cactusbot/api.py

```diff
@@ -154,6 +154,12 @@ class Command(CactusAPIBucket):
             self.api.patch("/user/{token}/command/{command}/count".format(
                 token=self.api.token, command=command), data=json.dumps(data)))
 
+    async def update(self, command, value):
+        """Update command attributes."""
+
+        return await self.api.patch("/user/{token}/command/{command}".format(
+            token=self.api.token, command=command), data=json.dumps(value))
```

Comment:
It would be nice if the type signature was based more on values, rather than just `json.dumps`ing `value`.

For example, something like:
```python
async def update(self, command, name=None, role=None):
```